### DISCUSSION TITLE
add support for multiple vaults

### DISCUSSION THREAD

**Discussion by ksdavidc** [coddingtonbear/obsidian-web#26] [Ideas]
I use different vaults on my system for different purposes. It would be nice if I could choose between them when downloading from a page, i.e. having several API keys available. For now i have to download to one and export that file to the other, which is inconvenient.

**Comment by coddingtonbear** [comment]
I'm personally right now of a mind that this tool is already a little too complicated for folks, and I don't have a personal use case for such a feature; so I'm afraid I'm not likely to implement that personally.

If you wanted to think about implementing such a feature on your own, though -- assuming we could come to an agreement about how this might work in the UI from a UX/UI and technical perspective -- I'd be glad to talk you through what a PR adding that feature would need to make it into the extension.

**Comment by ksdavidc** [comment]
Thanks for the reply. I understand your position. I am not sure my coding skills are up to that, but if I get desperate I will consider brushing up. Nonetheless, I appreciate the openness, and I do appreciate this plugin. It's helped me a lot in doing research for my classes. :-)
